src/ECharts/Formatter.purs

```diff
@@ -8,17 +8,19 @@ import Data.Argonaut.Core
 import Data.Argonaut.Encode
 import Data.Argonaut.Decode
 
-
 import Data.Function
 
 import Control.Monad.Eff
 
+
 type FormatParams = Json
 
 data Formatter =
   Template String
   | FormatFunc (forall eff. Array FormatParams -> Eff eff String)
```

Comment:
altered `FormatFunc (forall eff. Array FormatParams -> Eff eff String)` to `FormatFunc (Array FormatParams -> String)`